x-pack/plugins/security_solution/public/flyout/right/components/header_title.tsx

```diff
@@ -68,7 +69,14 @@ export const HeaderTitle: FC = memo(() => {
         </EuiFlexGroup>
       </EuiTitle>
       <EuiSpacer size="m" />
-      {timestamp && <PreferenceFormattedDate value={new Date(timestamp)} />}
+      <EuiFlexGroup direction="row" gutterSize="l">
+        <EuiFlexItem grow={false}>
+          <DocumentStatus />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          {timestamp && <PreferenceFormattedDate value={new Date(timestamp)} />}
+        </EuiFlexItem>
+      </EuiFlexGroup>
       <EuiSpacer size="m" />
```

Comment:
nit: the header in mock is more compact
PR:
![image](https://github.com/elastic/kibana/assets/18648970/e1a519fa-2709-4dcc-b09e-0e401c62094f)
mock:
![image](https://github.com/elastic/kibana/assets/18648970/52f8991d-4948-4388-a060-56271ad83731)